tests/Doctrine/Tests/DBAL/LegacyAssertions.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Doctrine\Tests\DBAL;
+
+trait LegacyAssertions
```

Comment:
So far, this trait implements modern assertions for legacy PHPUnit versions. The name looks like the opposite.